src/macros.rs

```diff
@@ -146,7 +181,8 @@ macro_rules! redis_module {
                 $firstkey:expr,
                 $lastkey:expr,
                 $keystep:expr,
-                $acl_categories:expr
+                $mandatory_command_acl_categories:expr
```

Comment:
Again, this is plural but represents just one category; IIRC, there is just one mandatory category now, or am I wrong? Either way, I understand the precaution here of actually passing more than one as a string separated with spaces, but I'd really like to have it more verbose and explicit by having the enum and a `[,]` expression instead then.

Reply:
there may be many mandatory categories. mandatory just means that if adding any of these categories fails, the module loading should fail.

Reply:
Alright, then  I'd like to have those not as string literals but as a comma-separated values list, which implements `From<T> for CommandAclCategory` enum either way, so either a string, or a `CommandAclCategory` enum variant itself.